README.md

```diff
@@ -15,6 +15,4 @@ connect(&cron, SIGNAL(activated()),   this, SLOT(onActivation()));
 connect(&cron, SIGNAL(deactivated()), this, SLOT(onDeactivation()));
 ```
 
-The 'activated' signal will be emitted every weekday at 8.00 and
-14.00, and the 'deactivated' signal will be emitted every weekday at
-12.00 and 18.00.
+The “Activated” signal will be given every weekday every minute from 8:00 to 11:59 and from 14:00 to 17:59.
```

Comment:
What about
```suggestion
The `activated` signal will be emitted every weekday every minute from 8:00 to 11:59 and from 14:00 to 17:59.
The `deactivated` signal will be emitted every weekday at 12.00 and 18.00.
```
* I'd rather keep the backquotes to indicate we're talking about the name of the signal in the code. There's no actual string involved here.
* "Emitted" is the term used in [Qt documentation](https://doc.qt.io/qt-6/signalsandslots.html).
* I appreciate the "every minute" precision +1
* I'd like to keep what happens with the `deactivated` signal